.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.1
```

Comment:
This is the .ruby-version which is for rbenv to work, i'm guessing you meant in .travis.yml?